Decred Wallet/Features/Receive/Receive.storyboard

```diff
@@ -0,0 +1,310 @@
+                                            </label>
+                                            <label opaque="NO" contentMode="left" text="(Tap to copy)" textAlignment="center" lineBreakMode="tailTruncation" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="KIw-9c-4KG">
+                                                <rect key="frame" x="0.0" y="437.5" width="398" height="20.5"/>
+                                                <fontDescription key="fontDescription" name="SourceSansPro-Regular" family="Source Sans Pro" pointSize="16"/>
+                                                <color key="textColor" red="0.53725490196078429" green="0.59215686274509804" blue="0.6470588235294118" alpha="1" colorSpace="custom" customColorSpace="sRGB"/>
+                                                <nil key="highlightedColor"/>
+                                                <userDefinedRuntimeAttributes>
+                                                    <userDefinedRuntimeAttribute type="string" keyPath="xibLocalizedStringKey" value="tapToCopy"/>
+                                                </userDefinedRuntimeAttributes>
+                                            </label>
+                                        </subviews>
+                                        <color key="backgroundColor" white="1" alpha="1" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
+                                    </stackView>
+                                </subviews>
+                                <color key="backgroundColor" systemColor="systemBackgroundColor" cocoaTouchSystemColor="whiteColor"/>
+                                <constraints>
+                                    <constraint firstItem="CjE-tq-m7A" firstAttribute="top" secondItem="dvn-e5-AOF" secondAttribute="top" id="PO9-zn-iHD"/>
+                                    <constraint firstAttribute="bottom" secondItem="CjE-tq-m7A" secondAttribute="bottom" constant="24" id="Shz-Mc-inr"/>
+                                    <constraint firstItem="CjE-tq-m7A" firstAttribute="leading" secondItem="dvn-e5-AOF" secondAttribute="leading" id="sym-VX-IYn"/>
+                                    <constraint firstAttribute="trailing" secondItem="CjE-tq-m7A" secondAttribute="trailing" id="yf2-Y4-dsp"/>
+                                </constraints>
+                                <userDefinedRuntimeAttributes>
+                                    <userDefinedRuntimeAttribute type="number" keyPath="borderRadius">
+                                        <real key="value" value="14"/>
+                                    </userDefinedRuntimeAttribute>
+                                </userDefinedRuntimeAttributes>
+                            </view>
+                            <view contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="Bxh-8Q-KbY">
```

Comment:
This view that holds the share button should be included in the `mainContentViewHolder`, so that it is hidden together with the other views when the sync in progress label is displayed.